spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/endpoint/web/PathMappedEndpoints.java

```diff
@@ -65,13 +65,13 @@ public PathMappedEndpoints(String basePath, Collection<EndpointsSupplier<?>> sup
 
 	private Map<EndpointId, PathMappedEndpoint> getEndpoints(Collection<EndpointsSupplier<?>> suppliers) {
 		Map<EndpointId, PathMappedEndpoint> endpoints = new LinkedHashMap<>();
-		suppliers.forEach((supplier) -> {
+		for (EndpointsSupplier<?> supplier : suppliers) {
```

Comment:
Why not replacing the both `forEach` loops?  if there is a benefit of doing that (I might think of performance reasons)

Reply:
There's often a GC impact when using streams, but I think it's pretty safe to use `forEach` here. I'm not sure why we'd want to change it.